### DISCUSSION TITLE
Stuck on "starting your workflow run"

### DISCUSSION THREAD

**Discussion by jsumners** [community/community#26581] [Actions]
<p>Is there something wrong with <a href="https://github.com/atlsynth/site-src/blob/master/.github/workflows/main.yml" rel="nofollow noopener">https://github.com/atlsynth/site-src/blob/master/.github/workflows/main.yml</a> ? I have not had any runs proceed past “starting your workflow run…” The first one, started over two hours prior to this posting, is still showing that message. And I don’t see any way to debug or stop these runs.</p>

**Comment by imjohnbo** [comment, marked as answer]
<p>Hi there! It looks like you have a small typo in ubuntu-latest :slightly_smiling_face:</p>

**Comment by jsumners** [comment]
<p>:man_facepalming: Awesome. Thank you for seeing my dumb mistake.</p>

**Comment by AstmDesign** [comment]
<p>Hello guys,<br>
Actually i’m facing the same issue,<br>
to know more about my details please follow the below link<br>
<a href="https://github.com/actions/setup-ruby/issues/26" rel="nofollow noopener">https://github.com/actions/setup-ruby/issues/26</a></p>

**Comment by Artalus** [comment]
<p>Had the exact same typo as you did :man_facepalming:.</p>
<p>Thank you and <a class="mention" href="/imjohnbo">@imjohnbo</a> for help. Github Actions should really notify you about those sorts of errors, instead of hanging forever with gray indicator!</p>

**Comment by jsumners** [comment]
<p>I agree. I definitely see this as a bug and hope someone on the team recognizes it as such.</p>

**Comment by mscoutermarsh** [comment]
<p><a class="mention" href="/jsumners">@jsumners</a> <a class="mention" href="/artalus">@artalus</a> Definitely. We have a couple improvements coming to help with this. Shouldn’t be this confusing when there is a typo!</p>

**Comment by jsumners** [comment]
<p>Excellent.</p>
<p>20 more characters to satisfy an arbitrary minimum.</p>

**Comment by NestorHernandezA** [comment]
<p>Am still having this issue, using one of the info dump snippets on here<a href="https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions#github-context" rel="nofollow noopener">https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions#github-context</a>. Was working before…</p>
<pre><code>name: "Context Information"

on: [pull_request]

jobs:
  info_job:
    name: Information
    runs-on: ubuntu-latest
    steps:
      - name: Dump GitHub context
        env:
          GITHUB_CONTEXT: ${{ toJson(github) }}
        run: echo "$GITHUB_CONTEXT"
      - name: Dump job context
        env:
          JOB_CONTEXT: ${{ toJson(job) }}
        run: echo "$JOB_CONTEXT"
      - name: Dump steps context
        env:
          STEPS_CONTEXT: ${{ toJson(steps) }}
        run: echo "$STEPS_CONTEXT"
      - name: Dump runner context
        env:
          RUNNER_CONTEXT: ${{ toJson(runner) }}
        run: |
          echo ${{ github.event.action.label.name }}
          echo "$RUNNER_CONTEXT"
      - name: Dump strategy context
        if: github.event.action.label.name == 'action-information'
        env:
          STRATEGY_CONTEXT: ${{ toJson(strategy) }}
        run: echo "$STRATEGY_CONTEXT"
      - name: Dump matrix context
        env:
          MATRIX_CONTEXT: ${{ toJson(matrix) }}
        run: echo "$MATRIX_CONTEXT"
</code></pre>

**Comment by marvinpinto** [comment]
<p>Similar situation for me. Jobs that used to work are currently stuck in the “Starting your workflow run…” state.</p>

**Comment by nhooyr** [comment]
<p>Same issue here.</p>

**Comment by steel** [comment]
<p>Same here as well. Tons of jobs queued.</p>

**Comment by steel** [comment]
<p>Is there any way to check github action service status?</p>

**Comment by aareet** [comment]
<p>I’m having trouble with this as well. I’m trying to automate movement of newly created issues into a project and the run is stick on “Starting your workflow run”. <a href="https://github.com/terraform-providers/terraform-provider-vsphere/blob/master/.github/workflows/issues.yml" rel="nofollow noopener">https://github.com/terraform-providers/terraform-provider-vsphere/blob/master/.github/workflows/issues.yml</a></p>

**Comment by jivoy1988** [comment]
<p>Same issue here.</p>
<p>Starting your workflow run…</p>

**Comment by j-keck** [comment]
<p>Same for me.</p>
<p>I tested all supported ubuntu versions:</p>
<p>‘ubuntu-latest’, ‘ubuntu-18.04’ and ‘ubuntu-16.04’.</p>
<p>The only version which works for me is ‘ubuntu-16.04’.</p>
<p>So i use ‘ubuntu-16.04’ as a workaround.</p>

**Comment by jivoy1988** [comment]
<p>I confirm.</p>
<p>The only version which works for me is ‘ubuntu-16.04’.</p>

**Comment by aqabawe** [comment]
<p>The issue with mine was specifying one job in an array as <code>needs</code>.<br>
<code>needs: [job1]</code> got my runner stuck until I replaced it with <code>needs: job1</code>.</p>

**Comment by Vadorequest** [comment]
<p>Same issue for me today. Except the script was working fine 15mn ago and there was no change in the script itself.</p>
<p>Looks like the GitHub Action just doesn’t start anymore.</p>

**Comment by MCOfficer** [comment]
<p>Same, it just takes a loooong while to get any runners today. Here’s hoping github’s on the case.</p>
